src/Build/BackEnd/Components/Scheduler/Scheduler.cs

```diff
@@ -144,7 +143,10 @@ internal class Scheduler : IScheduler
         /// <summary>
         /// Flag used for debugging by forcing all scheduling to go out-of-proc.
         /// </summary>
-        internal bool ForceAffinityOutOfProc { get; private set; }
+        internal bool ForceAffinityOutOfProc
+            => ChangeWaves.AreFeaturesEnabled(ChangeWaves.Wave17_0)
```

Comment:
On that note, can you also update https://github.com/dotnet/msbuild/blob/main/documentation/wiki/ChangeWaves.md?

Reply:
This matches the existing logic though. See the removed code below.

Reply:
To clarify, I moved this code from the ctor to the getter so that it's re-evaluated each time and not only once when the Scheduler is constructed. This is because the BuildParameters values may change over the lifetime of the Scheduler, so it needs to be read every time.

Reply:
You're right; ignore us.

Reply:
Ah I didn't catch that on the first go-round. Thanks for the clarification